app/models/oembed_error.rb

```diff
@@ -27,6 +28,13 @@ def add_error(error)
   # stacking up a bunch of the same error
   def unique_errors
     oembed_errors.map!(&:to_s).uniq!
-    touch if persisted?
+  end
+
+  def alert_depositor
+    # Only alert user if new errors are added
+    return unless saved_change_to_oembed_errors?
+
+    user = User.find_by_user_key(Generic.find(document_id).depositor)
```

Comment:
It might be better, here, to query Fedora directly:
`user = User.find_by_user_key(Generic.find(document_id).depositor)`
to 
`user = User.find_by_user_key(ActiveFedora::Base.find(document_id).depositor)`